README.md

```diff
@@ -54,7 +54,7 @@ myko ingests the events and can report:
 * The highest contributor origin of database load.
 * Events started at origins to help you analyze dependencies between services or components.
 
-See the [examples](tree/main/examples) directory for example programs.
+See the [examples](examples/) directory for example programs.
```

Comment:
Let's use the absolute path, https://github.com/rakyll/myko/tree/main/examples/.

The relative path depends on where the README is rendered. https://github.com/rakyll/myko and https://github.com/rakyll/myko/tree/main requires different relative paths.

Reply:
Done in e7d5cc9

Reply:
I made changes, but a point just came up in my mind. What if someone creates a new branch and makes changes in the `examples` directory? In that case, it always forwarded to the `examples` path in the `main` branch. Of course, it is not a big case, but it can be confusing for someone.

Reply:
Development branches are not for the users to browse, it's fine that links are just pointing to the main branch.  Thanks for the fix.